app/server.py

```diff
@@ -76,7 +76,42 @@ def index():
 
 @app.route('/inventory', methods=[GET])
 def query_prod_info():
-    """ Query specific entries in the Inventory system """
+    """
+    Retrieve a list of all the products in the inventory & query specific entries in the Inventory system
+    This endpoint will return all the details of the products in the inventory unless a query parameter is specificed
+    ---
+    tags:
+      -     Inventory
+    description: The inventory endpoint allows you to query the inventory
+    parameters:
+      -     name: prod_name
+            in: query
+            description: the name of the product you are looking for
+            required: false
+            type: string
+      -     name: quantity
+            in: query
+            description: if you want to check how many products have a specfic quantity
+            required: false
+            type: integer
+      -     name: condition
```

Comment:
Can you add an 'enum' attribute to "condition"? It would be clearer what exact conditions users can give.

Reply:
So you want me to change the type to enum?

Reply:
It's still a string. Just add an "enum" attribute.
enum:
  - new
  - used
  - open_boxed